code/tests/functional/tests/backend_api/default/test_conversation_custom.py

```diff
@@ -625,13 +626,9 @@ def test_post_makes_correct_call_to_store_conversation_in_search(
 def test_post_returns_error_when_downstream_fails(
     app_url: str, app_config: AppConfig, httpserver: HTTPServer
 ):
-    # given
-    httpserver.clear_all_handlers()  # Clear default successful responses
-
-    httpserver.expect_request(
-        "/indexes",
-        method="GET",
-    ).respond_with_json({}, status=500)
+    httpserver.expect_oneshot_request(
```

Comment:
The previous `clear_all_handlers` was not working, as it was clearing the call to mocked call to storage, which is now the first request made.

Instead of explicitly setting the `/config/active.json` request to fail, I changed it so any request will fail

I also changed the status code to 403, as the client was retrying 500s